packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -44,6 +44,7 @@ import {
 import { MCULayoutEventHandler } from './LayoutHandler';
 import Call from './Call';
 
+export const SDK_VERSION = '2.5.1';
```

Comment:
Hey @DeividVeloso , I have a question. 
Does this constant will change when running the github action when creating a new release?